cumulusci/cli/cci.py

```
@@ -163,6 +179,59 @@ def init_sentry():
+    # Interactive: ask the user
+    try:
+        console = Console(stderr=True)
+        console.print()
+        if click.confirm(
+            "Would you like to report this Salesforce error to Clariti to help improve CumulusCI?",
+            default=False,
+        ):
+            sentry_sdk.capture_exception(e)
+            console.print("[dim]Error report sent. Thank you.[/dim]")
+    except Exception:
+        pass  # stdin not available or other prompt failure - silently skip
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)